### PR TITLE
Update ASFF template to use label for severity

### PR DESCRIPTION
**Pull request by DavidWittman**
Use of the `Normalized` and `Product` fields is deprecated in the [ASFF spec](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format-attributes.html#asff-severity). Instead, we should just provide the severity as a `Label`, which simplifies the logic in this template quite a bit as well.

### CODE REVIEW

**Review by knqyf263 — COMMENTED** [review]
Thank you for the nice PR! I realized that we need to handle UNKNOWN. Trivy has 5 severities, UNKNOWN, LOW, MEDIUM, HIGH, CRITICAL, while Security Hub has INFORMATIONAL, LOW, MEDIUM, HIGH, CRITICAL. Could you map UNKNOWN to INFORMATIONAL?

**Comment by knqyf263** [comment]
@DavidWittman Also, you have to fix the integration test.

**Comment by DavidWittman** [comment]
Thanks for the feedback @knqyf263... I addressed both of the issues you mentioned. 👍  to having test coverage on the templates, I wasn't even considering to check for that.

File: contrib/asff.tpl [hunk 1]
```diff
@@ -37,8 +26,7 @@
         "CreatedAt": "{{ getCurrentTime }}",
         "UpdatedAt": "{{ getCurrentTime }}",
         "Severity": {
-            "Product": {{ $trivyProductSev }},
-            "Normalized": {{ $trivyNormalizedSev }}
+            "Label": "{{ .Severity }}"
```
Comment (knqyf263) [inline contrib/asff.tpl:29]: Looks like you forget to replace with `$severity`.

```suggestion
            "Label": "{{ $severity }}"
```

**Review by knqyf263 — APPROVED** [review]

File: contrib/asff.tpl (hunk 1 above)
Comment (knqyf263) [reply contrib/asff.tpl]: I've fixed it. Let me know if I'm wrong.

**Comment by knqyf263** [comment]
@DavidWittman Thanks!

**Pull request merged**